ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__multi-stable-openshift-logging-6.2.yaml

```diff
@@ -0,0 +1,101 @@
+      channel: candidate
+      version: "4.18"
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: aws-usgov-ipi-private-sts-f7-logging
+  cron: 29 6 * * 1
+  steps:
+    cluster_profile: aws-usgov-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "3"
+      EXTRACT_MANIFEST_INCLUDED: "true"
+      FILTERS_ADDITIONAL: ""
+      OO_INDEX: quay.io/openshift-qe-optional-operators/aosqe-index:log6.2
+      OPERATORS: "[   \n    {\"name\": \"loki-operator\", \"source\": \"qe-app-registry\",
+        \"channel\": \"stable-6.2\", \"install_namespace\": \"openshift-operators-redhat\",\"target_namespaces\":
+        \"\", \"operator_group\": \"openshift-operators-redhat\"},\n    {\"name\":
+        \"cluster-logging\", \"source\": \"qe-app-registry\", \"channel\": \"stable-6.2\",
+        \"install_namespace\": \"openshift-logging\",\"target_namespaces\": \"\",
+        \"operator_group\": \"openshift-logging\"},\n    {\"name\": \"amq-streams\",
+        \"source\": \"qe-app-registry\", \"channel\": \"stable\", \"install_namespace\":
+        \"openshift-operators\",\"target_namespaces\": \"\", \"operator_group\": \"\"}\n]\n"
+      TEST_ADDITIONAL: ""
+      TEST_FILTERS: CPaasrunOnly;CPaasrunBoth
+      TEST_IMPORTANCE: Critial|High
+      TEST_PARALLEL: "3"
+      TEST_SCENARIOS: sig-openshift-logging
+      TEST_TIMEOUT: "15"
+    test:
+    - ref: install-operators
+    - chain: openshift-e2e-test-logging-qe
+    workflow: cucushift-installer-rehearse-aws-usgov-ipi-private-cco-manual-security-token-service
+- as: azure-ipi-arm-f28-logging
+  cron: 5 15 28 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "2"
+      COMPUTE_NODE_TYPE: Standard_D4ps_v5
+      FILTERS_ADDITIONAL: ""
+      LOGGING_INDEX_IMAGE: quay.io/openshift-qe-optional-operators/aosqe-index:log6.2
+      OCP_ARCH: arm64
+      OPERATORS: |
+        [
+            {"name": "loki-operator", "source": "qe-app-registry", "channel": "stable-6.2", "install_namespace": "openshift-operators-redhat","target_namespaces": "", "operator_group": "openshift-operators-redhat"},
+            {"name": "cluster-logging", "source": "qe-app-registry", "channel": "stable-6.2", "install_namespace": "openshift-logging","target_namespaces": "", "operator_group": "openshift-logging"},
+            {"name": "cluster-observability-operator", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-cluster-observability-operator","target_namespaces": "", "operator_group": "openshift-cluster-observability-operator"},
+            {"name": "amq-streams", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-operators","target_namespaces": "", "operator_group": ""}
+        ]
+      TEST_ADDITIONAL: ""
+      TEST_FILTERS: Disruptive;Longduration
```

Comment:
@anpingli Since logging 6.2 targets OCP 4.18, I suggest to add a job in 4.18 to test `~Disruptive&;~Longduration&` cases, then we can cover all automated cases in OCP 4.18.

Reply:
I define two full-function by in different time in 28 days. 

4.17: - as: gcp-ipi-fips-f28-logging
  cron: 5 15 7 * *
4.18: azure-ipi-arm-f28-logging
   cron: 5 15 21 * *

Reply:
The job `azure-ipi-arm-f28-logging` doesn't run full function, it only runs `Disruptive` or `Longduration` cases.

Reply:
Yes.  two jobs run fullfunction together for 4.17 and 4.18.